src/main/java/org/dcsa/jit/controller/TimestampController.java

```diff
@@ -10,29 +10,25 @@
 import reactor.core.publisher.Mono;
 
 import javax.validation.Valid;
-import java.util.UUID;
 
 @RestController
 @RequiredArgsConstructor
 @RequestMapping(value = "timestamps", produces = {MediaType.APPLICATION_JSON_VALUE})
-public class TimestampController extends ExtendedBaseController<TimestampService, Timestamp, UUID> {
+public class TimestampController {
 
     private final TimestampService timestampService;
+    private final ObjectMapper objectMapper;
 
-    @Override
-    public TimestampService getService() {
-        return timestampService;
-    }
-
-    @Override
     public String getType() {
```

Comment:
This method can go too right? since we're not using `ExtendedBaseController` in this class anymore.

Reply:
Looks like you are right, removed. Thanks for spotting it. :)